eng/targets/Imports.targets

```diff
@@ -423,4 +423,51 @@
       <EnableNgenOptimization>true</EnableNgenOptimization>
     </PropertyGroup>
   </Target>
+
+  <!--
```

Comment:
This is reworked from Arcade's targets https://github.com/dotnet/arcade/blob/e84a945b04d1f83c7fc859e25b0e44d490697bc7/src/Microsoft.DotNet.Arcade.Sdk/tools/BuildReleasePackages.targets